src/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py

```diff
@@ -177,24 +181,29 @@
 import logging
 import time
 from collections import OrderedDict
-from glob import glob
+
+# from glob import glob
 from datetime import datetime
+from subprocess import PIPE
+
 
 import grass.script as gs
+from grass.pygrass.modules import Module
```

Comment:
Personally I prefer PyGRASS API. But what is the reason to use it on single place only?

Reply:
Done in  [37f7fd5](https://github.com/OSGeo/grass-addons/pull/1044/commits/37f7fd5ba7a690efef565e563e4486bed8383e4f)

Reply:
I wanted to pipe to _stdin_ of `m.proj` and use the results in `stdout` at the same time, see: 

https://github.com/OSGeo/grass-addons/blob/4a8e44e12eba593ad12916fd48d32470c3f2914c/src/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py#L282-L297

I am not aware of how to do that in the script API, while I knoew it to work in pygrass. If you have a suggestion for how to do this, I am happy to adjust...